core/src/main/java/de/gurkenlabs/litiengine/environment/CreatureMapObjectLoader.java

```diff
@@ -99,20 +91,35 @@ protected Creature createNewCreature(IMapObject mapObject, String spriteSheet) {
       }
     }
 
-    // if no custom creature type war registered for the spriteSheet, we just create a new Creature instance
+    // if no custom creature type war registered for the spriteSheet, we just create a new Creature
+    // instance
     return new Creature(spriteSheet);
   }
 
-  private static Creature createCustomCreature(Class<? extends Creature> customCreature, String spriteSheet) {
+  private static Creature createCustomCreature(
+      Class<? extends Creature> customCreature, String spriteSheet) {
     try {
       return customCreature.getConstructor(String.class).newInstance(spriteSheet);
-    } catch (InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException | SecurityException e) {
+    } catch (InstantiationException
+        | IllegalAccessException
+        | IllegalArgumentException
+        | InvocationTargetException
+        | NoSuchMethodException
+        | SecurityException e) {
       try {
         Creature creature = customCreature.getConstructor().newInstance();
         creature.setSpritesheetName(spriteSheet);
         return creature;
-      } catch (InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException | SecurityException ex) {
-        log.log(Level.WARNING, "Could not automatically create creature of type {0} because a matching constructor is missing.", new Object[] { customCreature });
+      } catch (InstantiationException
+          | IllegalAccessException
+          | IllegalArgumentException
+          | InvocationTargetException
+          | NoSuchMethodException
+          | SecurityException ex) {
+        log.log(
+            Level.WARNING,
+            "Could not automatically create creature of type {0} because a matching constructor is missing.",
+            new Object[] {customCreature});
```

Comment:
I see no reason for the catch clause to be this long, having it on one line makes it more readable. Same with the log statement.